tensorflow_io/core/BUILD

```diff
@@ -7,6 +7,19 @@ load(
     "tf_io_copts",
 )
 load("@io_bazel_rules_go//go:def.bzl", "go_binary")
+load("@bazel_skylib//rules:common_settings.bzl", "bool_flag")
+
+bool_flag(
+    name = "static_build",
+    build_setting_default = False,
+)
+
+config_setting(
+    name = "static_build_on",
+    flag_values = {
+        ":static_build": "True",
```

Comment:
@yongtang shouldn't this be a bool?

Reply:
@kvignesh1420 I tried before and it looks like the `flag_values` will take a `key:value` of `string:string`.